CONTRIBUTING.md

```diff
@@ -21,25 +31,60 @@ $ git add my_new_directory/  # Add an entire directory
 
-Once you are ready for others to see your changes, you should push them to GitHub. If you are a member of IEEE UofT, you can be granted limited push access to this repository, otherwise [create a fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo).
+Once you are ready for others to see your changes, you should push them to GitHub. If you are a member of IEEE UofT, you can be granted limited push access to this repository, and can open a pull request directly from your branch into `develop`. External contributors must make the pull request from their forked repository.
 ```bash
 $ git push -u origin my_new_feature
 ```
 
 ### Pull Requests
-Once your changes have been pushed, [create a pull request](https://github.com/ieeeuoft/hackathon-template/compare) with your changes into the `develop` branch. Give a descriptive title, and a description of the changes __including__ how to test them.
+Once your changes have been pushed, [create a pull request](https://github.com/ieeeuoft/hackathon-template/compare) with your changes into the `develop` branch. Give a descriptive title, and a description of the changes __including__ how to test them. Fill out everything in the pull request template, being as descriptive as possible and using appropriate markdown code formatting where necessary.
 
 Pull requests must be approved by members of the IEEE UofT web team. Reviewers must carefully read through all the changes made, and run them locally. They will add comments to your pull request with any changes they'd like to see or suggestions they have. 
 
+Pull requests must also pass the checks in our [CI workflow](https://github.com/ieeeuoft/hackathon-template/blob/develop/.github/workflows/main.yml). These include unit tests and code formatting checks. If a CI run fails for your pull request, please see why and make the necessary changes. A reviewer may not look at your pull request until checks are passing, unless you ask for help or feedback on something that's a work in progress.
+
 Once your pull request has been approved, our policy is that the **members of IEEE UofT should merge their own pull requests** as a final sign off on the changes. Pull requests from outside will be merged by a member of our team. Merges into `develop` should only be done with the "Squash and Merge" button.
 
+#### Checking Out Pull Requests
+For pull requests with branches committed directly to the repository, a regular `git checkout <branch name>` will work. Since branches should follow the above naming guidelines, this is usually as easy as doing `git pull` followed by `git checkout <issue number>-<TAB>`.
+
+ For pull requests from external repositories, you can still check them out from the main repository using the pull request number. For pull request `123`, do:
```

Comment:
Either change 123 to 226 here or vice versa in the code below

Reply:
Copy-pasting strikes again :man_facepalming: